src/lib/PageTransition.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+  import { fly } from "svelte/transition";
+  export let refresh = "";
+</script>
+
+{#key refresh}
+  <div
+		in:fly={{ y: 6, duration: 400, delay: 400 }}
```

Comment:
This doesn't respect the `prefers-reduced-motion` setting.

Reply:
I never work with `prefers-reduced-motion`, so🤷‍♂️ There isn't a built-in solution, and adding extra libraries and stuff is just making me say whatever, I didn't even want animations anymore😄